packages/Webkul/Checkout/src/Cart.php

```diff
@@ -120,6 +120,19 @@ public function addProduct($productId, $data)
             return ['info' => __('shop::app.checkout.cart.item.inactive-add')];
         }
 
+        if (isset($data['qty'])) {
```

Comment:
This can't be added here as by default 1 quantity is going if someone is able to bypass zero. This can be handled from the front end or otherwise handle for each product type individually.